distributed/cli/tests/test_dask_scheduler.py

```diff
@@ -255,3 +254,49 @@ def test_bokeh_port_zero(loop):
                 if b'bokeh' in line.lower() or b'web' in line.lower():
                     count += 1
                     assert b':0' not in line
+
+
+PRELOAD_TEXT = """
+from distributed.scheduler import Scheduler
+
+def dask_setup(scheduler):
+    if isinstance(scheduler, Scheduler):
+        print("setup")
+
+def dask_teardown(scheduler):
+    if isinstance(scheduler, Scheduler):
+        print("teardown")
+"""
+
+
+def test_preload_file(loop):
```

Comment:
Any idea why this fails on windows? I have no experience with Python on Windows. The Travis build is fine, one job failed and it seems to be unrelated (and non-deterministic).

https://ci.appveyor.com/project/daskdev/distributed/build/1.0.1040#L1412